use_cases/attachment.md

```diff
@@ -0,0 +1,45 @@
+# Attachment
+
+```python
+import base64
+import sendgrid
+import os
+from sendgrid.helpers.mail import Email, Content, Mail, Attachment
+try:
+    # Python 3
+    import urllib.request as urllib
+except ImportError:
+    # Python 2
+    import urllib2 as urllib
+
+sg = sendgrid.SendGridAPIClient(apikey=os.environ.get('SENDGRID_API_KEY'))
+from_email = Email("test@example.com")
+subject = "subject"
+to_email = Email("to_email@example.com")
+content = Content("text/html", "I'm a content example")
+
+file_path = "file_path.pdf"
+with open(file_path,'rb') as f:
+    data = f.read()
+    f.close()
```

Comment:
The `with` statement context manager handles closing the file automatically, is this, `f.close()`, really necessary?